sky/serve/serve_state.py

```diff
@@ -34,7 +34,7 @@ def _get_db_path() -> str:
 def create_table(cursor: 'sqlite3.Cursor', conn: 'sqlite3.Connection') -> None:
     """Creates the service and replica tables if they do not exist."""
 
-    # auto_restart column is deprecated.
+    # auto_restart adn requested_resources column is deprecated.
```

Comment:
Should we also remove the `auto_restart` and `requested_resources` columns when creating the table?

Reply:
That would break the logic here:

https://github.com/skypilot-org/skypilot/blob/ee708e7aad508fab1b48b1f6f297ebe6fb5dbe63/sky/serve/serve_state.py#L324-L348

I think managed spot also keeps those columns for similar reasons:

https://github.com/skypilot-org/skypilot/blob/ee708e7aad508fab1b48b1f6f297ebe6fb5dbe63/sky/jobs/state.py#L41-L69